test/unary-tests.txt

```diff
@@ -67,25 +67,6 @@ UnaryTests(PositiveUnaryTests(
 ))
 
 
-# QualifiedName { "top": "UnaryTests" }
```

Comment:
Let's not remove this, but fix this instead.

I.e. rename this to `Interval`.

Reply:
Done with 08d3b9ed6c70adf0f061e49640f5df53e65f6eb8